app/controllers/pxe_controller/pxe_image_types.rb

```diff
@@ -60,19 +54,6 @@ def pxe_image_type_edit
     end
```

Comment:
Basically you will only need this part above, without the if-else logic around. Also you can get rid of the `add_flash` and `pxe_image_type_set_form_vars` and `session[:changed]` lines. This also means that you can drop the definition `pxe_image_type_set_form_vars` method completely.